src/Symfony/Component/Security/Guard/GuardAuthenticatorInterface.php

```diff
@@ -28,6 +28,8 @@
  * one location.
  *
  * @author Ryan Weaver <ryan@knpuniversity.com>
+ *
+ * @deprecated Symfony\Component\Security\Guard\AuthenticatorInterface must be used instead
```

Comment:
Do we need to `trigger_error` inside this file as well (and if so, we would need to copy this interface into `AuthenticatorInterface` so that using the new interface doesn't trigger the warning). I'm not sure because, we WILL trigger the deprecation notice the minute you try to use this interface in the guard system... so in practice... it's covered.

Reply:
AFAIK we don't, the Debug component triggers automatically based on the `@deprecated` annotation